docs/.vuepress/components/HelloWorld.vue

```diff
@@ -0,0 +1,59 @@
+<template>
+  <div> 
+    <tabs :options="{ useUrlFragment: false }"  @changed="tabClicked">
+      <tab name="JavaScript" id="js">
+        <iframe v-if="selected === 'js'" :src="jsSrc"
+          :style="styleAttribute"
+          :allow="allowAttribute"
+          :sandbox="sandboxAttribute"
+          title="Handsontable JavaScript Data Grid - Hello World App"
+        ></iframe>
+      </tab>
+      <tab name="TypeScript" id="typescript">
+        <iframe v-if="selected === 'typescript'" :src="tsSrc" 
+          :style="styleAttribute"
+          :allow="allowAttribute"
+          :sandbox="sandboxAttribute"
+          title="Handsontable TypeScript Data Grid - Hello World App"
+        ></iframe>
+      </tab>
+      <tab name="React" id="react">
+        <iframe v-if="selected === 'react'" :src="reactSrc"
+          :style="styleAttribute"
+          :allow="allowAttribute"
+          :sandbox="sandboxAttribute"
+          title="Handsontable React Data Grid - Hello World App"
+        ></iframe>
+      </tab>
+      <tab name="Vue" id="vue" :is-disabled="true"></tab>
+      <tab name="Angular" id="angular" :is-disabled="true"></tab>
+   </tabs>
+  </div>
+</template>
+
+<script>
+const hrefStart='https://codesandbox.io/embed/';
+const hrefEnd='?fontsize=14&hidenavigation=1&theme=dark&view=split&runonclick=1';
+
+export default {
+  name: 'HelloWorld',
+  data() {
+    return {
+      selected: 'js',
+      jsSrc: `${hrefStart}handsontable-javascript-data-grid-hello-world-app-dzx8f${hrefEnd}`,
+      tsSrc: `${hrefStart}handsontable-typescript-data-grid-hello-world-app-145es${hrefEnd}`,
+      reactSrc: `${hrefStart}handsontable-react-data-grid-hello-world-app-yt46w${hrefEnd}`,
```

Comment:
I propose extract creating url logic into a function:


```js
const getCsUrl = (id) => `${hrefStart}${id}${hrefEnd}`


// then
jsSrc: getCsUrl('handsontable-javascript-data-grid-hello-world-app-dzx8f'),
tsSrc: getCsUrl('handsontable-typescript-data-grid-hello-world-app-145es'),
reactSrc: getCsUrl('handsontable-react-data-grid-hello-world-app-yt46w'),
```

Because it will be much easier to maintain.

WDYT?

Reply:
Nice catch. I did it within: https://github.com/handsontable/handsontable/pull/8173/commits/ffa0609cf573827bb1d8be5a80a7b743f3b56d5a.

PS. Maybe it will be not necessary when your second suggestion will be applied.

Reply:
> PS. Maybe it will be not necessary when your second suggestion will be applied.

You are right, Thank you